guides/common/modules/proc_rex-pull-based-transport.adoc

```diff
@@ -1,8 +1,7 @@
 [id="REX_pull_based_transport_{context}"]
 = REX pull-based transport
 
-{Project} has a pull-based transport that has the features of REX but with a deployment model similar to katello-agent. 
-This transport uses MQTT as its messaging protocol and includes an MQTT client running on each host.
+{Project} uses MQTT as its messaging protocol and includes an MQTT client running on each host.
```

Comment:
I think it should still state that it's a pull based transport. By default there is on REX using SSH, which is push based. MQTT is entirely optional, but this phrasing suggests otherwise.